documentation/book/proc-restricting-access-to-listeners-using-network-policies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-restricting-access-to-listeners-using-network-policies-{context}']
+= Restricting access to Kafka listeners using NetworkPolicies
+
```

Comment:
_As per modular docs guidelines, procedures must have an introduction:_

You can restrict access to a listener to only selected applications by using the `networkPolicyPeers` field.